### ISSUE TITLE
Building Docker image with new version of Bun crashes at installation step

### ISSUE THREAD

**Issue by e965** [oven-sh/bun#15906]
### How can we reproduce the crash?

Everything is fine with version 1.1.40

Project dependencies:

![Image](https://github.com/user-attachments/assets/0a389fd5-c821-49dc-88a2-96704b25a757)

### Relevant log output

```shell
[+] Building 3.9s (9/15)                                                                                                                                                               docker:default
 => [project_web internal] load build definition from Dockerfile                                                                                                                                    0.0s
 => => transferring dockerfile: 629B                                                                                                                                                             0.0s
 => [project_web internal] load metadata for docker.io/oven/bun:1.1.41-alpine                                                                                                                       1.5s
 => [project_web internal] load .dockerignore                                                                                                                                                       0.0s
 => => transferring context: 194B                                                                                                                                                                0.0s
 => [project_web internal] load build context                                                                                                                                                       0.0s
 => => transferring context: 2.33kB                                                                                                                                                              0.0s
 => [project_web base 1/1] FROM docker.io/oven/bun:1.1.41-alpine@sha256:62dfb6c99d5e028a29c2ad44916d8071ad24c4d3965f94051a111201cf8904d4                                                            0.0s
 => CACHED [project_web runner 1/5] WORKDIR /project/dist                                                                                                                                              0.0s
 => CACHED [project_web builder 1/5] WORKDIR /build                                                                                                                                                 0.0s
 => CACHED [project_web builder 2/5] COPY . .                                                                                                                                                       0.0s
 => ERROR [project_web builder 3/5] RUN bun install                                                                                                                                                 2.2s
------                                                                                                                                                                                                
 > [project_web builder 3/5] RUN bun install:                                                                                                                                                            
0.412 bun install v1.1.41 (b8f28ed8)                                                                                                                                                                  
2.053 ============================================================                                                                                                                                    
2.053 Bun v1.1.41 (b8f28ed8) Linux x64 (baseline)
2.053 Linux Kernel v5.15.153 | musl
2.053 CPU: sse42 popcnt avx avx2
2.053 Args: "node" "scripts/postinstall.js"
2.053 Features: jsc tsconfig tsconfig_paths 
2.053 Builtins: "bun:main" "node:fs" "node:path" "node:string_decoder" "node:util/types" 
2.053 Elapsed: 101ms | User: 43ms | Sys: 21ms
2.053 RSS: 1.07GB | Peak: 0.17GB | Commit: 1.07GB | Faults: 226
2.053 
2.053 panic(main thread): Segmentation fault at address 0x0
2.053 Crashed while visiting /build/node_modules/@prisma/get-platform/dist/chunk-6HZWON4S.js
2.053 oh no: Bun has crashed. This indicates a bug in Bun, not your code.
2.053 
2.053 To send a redacted crash report to Bun's team,
2.053 please file a GitHub issue using the link below:
2.053 
2.053  https://bun.report/1.1.41/Bn1b8f28edChgg8//D4m+zkE0+vT0yv0nDi906hD2/yohD+73tgD0i6sgDugnugDs91p8D_A2AA
2.053 
2.053 
2.054 error: postinstall script from "@prisma/engines" terminated by SIGILL (Illegal instruction)
------
failed to solve: process "/bin/sh -c bun install" did not complete successfully: exit code: 139
```

### Stack Trace (bun.report)

Bun v1.1.41 ([`b8f28ed`](<https://github.com/oven-sh/bun/tree/b8f28ed8afd1c2b60568b2b0158d39a674178027>)) on linux x86_64_baseline [RunAsNodeCommand]

Segmentation fault at address 0x00000000

- [`ld-temp.o:0`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/ld-temp.o#L0>): `WTF::URL::URL`
- `??`
- [`array_list.zig:849`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027//opt/homebrew/var/buildkite-agent/builds/darwin-aarch64-studio-2-7/bun/bun/vendor/zig/lib/std/array_list.zig#L849>): `src.js_parser.NewParser_`
- [`watcher.zig:805`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/watcher.zig#L805>): `Thread.PosixThreadImpl.spawn__anon_123928.Instance.entryFn`
- [`bindings.zig:5864`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/bindings/bindings.zig#L5864>): `Bun__HTTPRequestContextDebugTLS__onReject`
- [`module_loader.zig:2523`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/module_loader.zig#L2523>): `src.bun.js.module_loader.ModuleLoader.fetchBuiltinModule`
- [`bindings.zig:654`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/bindings/bindings.zig#L654>): `src.bun.js.module_loader.ModuleLoader.fetchBuiltinModule`
- [`module_loader.zig:97`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/module_loader.zig#L97>): `src.bun.js.module_loader.ModuleLoader.fetchBuiltinModule`
- [`StringImpl.h:1161`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/./build/release-ci/cache/webkit-3845bf370ff4e9a5c0b96036255142c7904be963/include/wtf/text/StringImpl.h#L1161>): `WebCore::BunBuiltinNames::~BunBuiltinNames`

Features: tsconfig\_paths, jsc, tsconfig, tsconfig

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway in #15911

**Comment by e965** [comment]
@Jarred-Sumner 

I confirm that the problem has been fixed. Thank you!
